inc/thorvg.h

```diff
@@ -1339,7 +1339,17 @@ class TVG_EXPORT Initializer final
 /**
  * @class Saver
  *
- * @brief A class enabling saving a paint in a binary format.
+ * @brief A class exporting a paint in a specified file data to recover the paint scene later.
```

Comment:
I would write:
A class for exporting a paint object into a specified file, from which to recover the paint data later.

Reply:
updated.